app/controllers/api/v1/webauthn_verifications_controller.rb

```diff
@@ -2,21 +2,18 @@ class Api::V1::WebauthnVerificationsController < Api::BaseController
   def create
     authenticate_or_request_with_http_basic do |username, password|
       user = User.authenticate(username.strip, password)
+      next false if user.nil?
 
-      if user
-        if user.webauthn_credentials.present?
-          verification = user.refresh_webauthn_verification
-          webauthn_path = "example.com/webauthn/#{verification.path_token}"
-          respond_to do |format|
-            format.any(:all) { render plain: webauthn_path }
-            format.yaml { render yaml: { path: webauthn_path, expiry: verification.path_token_expires_at.utc } }
-            format.json { render json: { path: webauthn_path, expiry: verification.path_token_expires_at.utc } }
-          end
-        else
-          render plain: t("settings.edit.no_webauthn_credentials"), status: :unprocessable_entity
+      if user.webauthn_credentials.present?
+        verification = user.refresh_webauthn_verification
+        webauthn_path = "example.com/webauthn/#{verification.path_token}"
+        respond_to do |format|
+          format.any(:all) { render plain: webauthn_path }
+          format.yaml { render yaml: { path: webauthn_path, expiry: verification.path_token_expires_at.utc } }
+          format.json { render json: { path: webauthn_path, expiry: verification.path_token_expires_at.utc } }
         end
       else
-        false
```

Comment:
Could we do if `user&.webauthn_credentials`, then do an `elsif user; render plain....` `else; false`? Would that return the same thing?

Reply:
I assume it would, yes. I looked up [the definition for Cognitive Complexity](https://www.sonarsource.com/docs/CognitiveComplexity.pdf) and it explicitly ignores null-coalescing operators when computing the score. I'll amend to that approach.